source/classes.tex

```diff
@@ -5266,9 +4804,10 @@
 \pnum
 Such a function is implicitly an inline\iref{dcl.inline} function
 if it is attached to the global module.
-A friend
-function defined in a class is in the (lexical) scope of the class in which it is defined.
-A friend function defined outside the class is not\iref{basic.lookup.unqual}.
+\begin{note}
+If a friend function is defined outside a class,
+it is not in the scope of the class.
```

Comment:
Does " the scope of the class" here mean "the target scope of the class" or "the scope established by the class"?

Reply:
Deferring.

Reply:
It means the latter; "the scope of _X_" is used throughout to mean that introduced by _X_ (which might be an entity or a grammar production like a *compound-statement*) or its declaration(s).  We could of course define an explicit term for it in [basic.scope.scope] if desired.